translation-language-detection-api-playground/index.html

```diff
@@ -46,11 +34,13 @@ <h1>💬 Translator and Language Detector API Playground</h1>
         >Translate to
         <select id="translate">
           <option value="en">English</option>
+          <option value="fr">French</option>
           <option value="ja">Japanese</option>
+          <option selected value="pt">Portuguese</option>
           <option selected value="es">Spanish</option>
         </select></label
       >
-      <button hidden type="submit">Translate</button>
+      <button hidden type="submit" class="button-style">Translate</button>
```

Comment:
FWIW I get `Currently, only English ↔ Spanish and English ↔ Japanese are supported.` message when it's not supported. We should communicate this to the Chrome team if not already.

Reply:
@alexandrascript Did you start a conversation already?